cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,26 +1,39 @@
+import {MenuContentPage} from "../page/menu-content.page";
+import {TShirtsPage} from "../page/t-shirts.page";
+import {SignInPage} from "../page/sign-in.page";
+import {AddressPage} from "../page/address.page";
+import {ShippingPage} from "../page/shipping.page";
+import {PaymentPage} from "../page/payment-step.page";
+
+const menuContentPage = new MenuContentPage();
+const tShirtsPage = new TShirtsPage();
```

Comment:
This should be called ProductiListPage
![Screen Shot 2022-06-07 at 12 40 49 PM](https://user-images.githubusercontent.com/36927617/172447703-399afae0-c2ac-4604-aa25-3a310c27b52e.png)